app/containers/UIKit/MultiSelect/index.tsx

```diff
@@ -71,43 +47,46 @@ export const MultiSelect = React.memo(
 		inputStyle,
 		innerInputStyle
 	}: IMultiSelect) => {
-		const { theme } = useTheme();
+		const { theme, colors } = useTheme();
 		const [selected, select] = useState<any>(Array.isArray(values) ? values : []);
-		const [open, setOpen] = useState(false);
-		const [search, onSearchChange] = useState('');
 		const [currentValue, setCurrentValue] = useState('');
-		const [showContent, setShowContent] = useState(false);
+
+		const { showActionSheet, hideActionSheet } = useActionSheet();
 
 		useEffect(() => {
 			if (Array.isArray(values)) {
 				select(values);
 			}
 		}, [values]);
 
-		useEffect(() => {
-			setOpen(showContent);
-		}, [showContent]);
-
 		useEffect(() => {
 			if (values && values.length && !multiselect) {
 				setCurrentValue(values[0].text);
 			}
 		}, []);
 
 		const onShow = () => {
-			Animated.timing(animatedValue, {
-				toValue: 1,
-				...ANIMATION_PROPS
-			}).start();
-			setShowContent(true);
+			showActionSheet({
+				children: (
+					<MultiSelectContent
+						options={options}
+						onSearch={onSearch}
+						theme={theme}
+						select={select}
+						onChange={onChange}
+						setCurrentValue={setCurrentValue}
+						onHide={onHide}
+						multiselect={multiselect}
+						selectedItems={selected}
+					/>
+				),
+				snaps: [300],
```

Comment:
I'm not sure we need this

Reply:
We do need to specify the snaps here as we're using the custom component logic but we'll surely want multiple snaps for longer FlatLists. I did this just to match the previous design and will update the logic here.